src/convertToObject.js

```diff
@@ -3,16 +3,28 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
+ * and returns an object where
+ * CSS properties are keys
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const obj = {};
+  let arr = sourceString.split(/[:;\n]/);
```

Comment:
why is it `let`? you just change it with a `filter` which is non-mutating method, make it a const and you can actually chain methods together - look at [checklist](https://github.com/mate-academy/js_style-to-object/blob/master/checklist.md) 4th item to find out how

Reply:
also, `arr` is a bad name for a variable